### DISCUSSION TITLE
당신의 인생이 왜 힘들지 않아야 한다고 생각하십니까: 쇼펜하우어 아포리즘 [인내]

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#537] [4. 기타 글 모음]
요즘 고민이 참 많습니다!! 갑작스럽게 회사에서 한 제안을 받았는데 미래에 과연 도움이 될 선택인지 아닐지 계속 고민하고 있는데 쉽사리 정답이 나오지 않네요..

그동안 미뤄왔던 독서와 대학교 입학하기 전에 꼭 해보고 싶었던 경제학 공부(지금은 미시경제를 보고 있습니다!)를 하고 있습니다! 이참에 공부하면서 자격증도 재미삼아 몇개 따보려고 합니다! 백엔드 관련해서 글을 써야하는데 ㅎㅎ;; 기,,기억보단 기록을 해보려고 노력하겠심다.

최근에 쇼펜하우어의 명언과 같은 글들을 여러개 모아둔 책을 읽고 있는데 그냥 좀 최근에 제일 제게 필요한 글들만 두가지 정도 기록해놨습니다.
[게시물 바로가기](https://stopmin.tistory.com/entry/%EB%8B%B9%EC%8B%A0%EC%9D%98-%EC%9D%B8%EC%83%9D%EC%9D%B4-%EC%99%9C-%ED%9E%98%EB%93%A4%EC%A7%80-%EC%95%8A%EC%95%84%EC%95%BC-%ED%95%9C%EB%8B%A4%EA%B3%A0-%EC%83%9D%EA%B0%81%ED%95%98%EC%8B%AD%EB%8B%88%EA%B9%8C-%EC%87%BC%ED%8E%9C%ED%95%98%EC%9A%B0%EC%96%B4-%EC%95%84%ED%8F%AC%EB%A6%AC%EC%A6%98-%EC%9D%B8%EB%82%B4)

**Comment by 1jeongg** [comment]
멋있네요 👍👍

**Reply by stopmin** [reply]
원정님도 멋있으십니다!! 👍👍👍 같이 파이팅해요!!
